### DISCUSSION TITLE
Please add Rohanasan backend framework

### DISCUSSION THREAD

**Discussion by RohanVashisht1234** [TechEmpower/FrameworkBenchmarks#8813] [Ideas]
**Subject: Request for Inclusion of Rohanasan Framework in Next Round of Tech Empowerment**
Link for rust release:
https://docs.rs/rohanasan/latest/rohanasan/

I hope this message finds you well. I am writing to propose the inclusion of the Rohanasan framework for the next round of Tech empowerment.

Rohanasan, as evidenced by its documentation and performance benchmarks, is a fast back-end framework developed in the Rust programming language.

I kindly request the consideration of this proposal for inclusion in the upcoming round of tech empowerment initiatives.

Best regards,
Rohan Vashisht
Developer and Maintainer of Rohanasan Framework

**Comment by joanhey** [comment]
You are welcome to add this framework, as the benchmark is open source.
